TESTS/mbed_hal/spm/main.cpp

```diff
@@ -23,8 +23,8 @@
 #error [NOT_SUPPORTED] this test is supported on GCC only
 #endif
 
-#if defined(TARGET_FUTURE_SEQUANA_PSA)
-#error [NOT_SUPPORTED] Disable this Test until FUTURE_SEQUANA_PSA enables Memory protection
+#if defined(TARGET_FUTURE_SEQUANA_PSA) || defined(TARGET_CY8CKIT_062_WIFI_BT_PSA)
```

Comment:
🎉

Reply:
Not yet. Our tests are still running

Reply:
Enable this test please
Your platform has memory separation enabled

Reply:
done